### PR TITLE
Feat: Added video listing page

### PR DESCRIPTION
**Pull request by vinc3nati**
- sidebar
- video list
- responsive

image

### CODE REVIEW

File: src/components/Chips/Chips.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  useEffect(() => {
+    filters.category === categoryName ? setActive(true) : setActive(false);
+    filters.category === "" && categoryName === "all" && setActive(true);
+  }, [filters.category, categoryName]);
+
+  return (
+    <div
+      className={`chip ${active && "chip-active"}`}
+      onClick={() => {
+        if (!active)
```
Comment (shrey27) [inline src/components/Chips/Chips.jsx]: put this code in a separate functions instead of inline functions

**Review by shrey27 — APPROVED** [review]
code structure is good
UI looks great
PR is ready to merge.

**Review by anuj-kumary — COMMENTED** [review]
Clean code and good use of naming convention
Good to merge

**Review by anuj-kumary — APPROVED** [review]

File: src/components/Chips/chips.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    background-color: var(--color-background-accent);
+    color: var(--text-color);
+    padding: 0.7rem 1rem;
+    cursor: pointer;
+    border-radius: var(--border-radius-14);
+    -webkit-border-radius: var(--border-radius-14);
+    -moz-border-radius: var(--border-radius-14);
+    -ms-border-radius: var(--border-radius-14);
+    -o-border-radius: var(--border-radius-14);
+}
```
Comment (OmKakatkar) [inline src/components/Chips/chips.css]: like the use of browser prefixes

**Review by OmKakatkar — APPROVED** [review]
Overall looks good.
Good variable names.
Clean and Modular code

**Pull request merged**